index.html

```diff
@@ -166,8 +167,36 @@ <h3>Skills</h3>
                 </li>  
             </ul>
           
-        </section>       
+        </section> 
+
     </main> 
- 
+    
+    <footer> 
+        <p class="primary-text">
+            I'm always interested in hearing about new projects,
+             so if you'd like to chat please get in touch.</p>
+
+        <form action="https://formspree.io/f/xpzbzzjw" method="POST">
```

Comment:
- [ ] Your design is working well 👍; the requirements want you to use `flexbox` for this section. please layout these elements using `flexbox`. this will help you practice using `flexbox`.